.docker/testing/docker-compose-travis.yml

```
@@ -6,4 +6,4 @@ services:
       dockerfile: ./.docker/testing/Dockerfile
     tty: true
     volumes:
-      - ${TRAVIS_BUILD_DIR}:/usr/src
+      - /home/runner/work/QField/QField:/usr/src
```

I would suggest we add  gh workflows as a parallel build engine while we keep travis in place for now. Travis has also all the deployment included which will take a bit of time to migrate as well.

I would rewrite all the travis specific env vars to generic ones (e.g. with a `CI` prefix) and set them in .travis.yml  and the gh workflows to match the environment. This way we can move gradually.

```suggestion
      - ${CI_BUILD_DIR}:/usr/src
```

---

I am experimenting with this now, seems promissing. All the variables are `CI_` prefixed and set by calling `env_travis.sh` or `env_gh.sh` respectively. I use the other `lupdate` for experimenting to reduce the noise here, and I will commit once all is working again.

---

Done! All travis specific vars are now `$CI_*` vars.